packages/html-reporter/src/testResultView.tsx

```diff
@@ -70,7 +70,7 @@ export const TestResultView: React.FC<{
   const { screenshots, videos, traces, otherAttachments, diffs, errors, htmls } = React.useMemo(() => {
     const attachments = result?.attachments || [];
     const screenshots = new Set(attachments.filter(a => a.contentType.startsWith('image/')));
-    const videos = attachments.filter(a => a.name === 'video');
+    const videos = attachments.filter(a => a.contentType === 'video/webm');
```

Comment:
We should allow all video types:

```suggestion
    const videos = attachments.filter(a => a.contentType.startsWith('video/'));
```

Reply:
fair point, I wanted to be conservative in order not to introduce regression but if you say that all video types are supported I'm ok with your suggestion.